src/protocols/aeternity/config.ts

```diff
@@ -151,6 +156,9 @@ export const TX_FUNCTIONS_TYPE_DEX: Record<DexFunctionType, TxFunctionRaw[]> = {
   addLiquidity: [
     'add_liquidity', 'add_liquidity_ae',
   ],
+  provideLiquidity: [
+    'provide_liquidity', 'provide_liquidity_ae',
+  ],
```

Comment:
@CedrikNikita can you take a look on this? Is this a real TxFunction value?

Reply:
This is the real name of the dex functions.
https://github.com/aeternity/dex-contracts-v2/blob/2eabf1c3e6a4f85f85afaa9409259ae72d0c75b6/contracts/interfaces/router/IAedexV2Router.aes#L24-L62

This is map that Pawel came up with back then.
https://github.com/superhero-com/superhero-wallet/commit/f39e1cda7ebe06b49ea8f9c20a4a834b3863a7b1#diff-c0370e3ffa17358bef68988fb764a8bdf758479058bf9975be7eb0607840ed9cR54-R55

Reply:
OK. I'll remove this.